tools/boilerplate.py

```diff
@@ -365,7 +365,7 @@ def format_value(value):
 
 
 def build_pyplot():
-    pyplot_path = os.path.join(os.path.dirname(__file__), 'lib',
+    pyplot_path = os.path.join("../", os.path.dirname(__file__), 'lib',
```

Comment:
This doesn't look quite right; if `__file__` is an absolute path, then it'll ignore the `..` part. Less importantly, there shouldn't be a trailing `/` since it's using `os.path.join`. 

It should probably be `os.pardir` and come after the `dirname`.